actions/license-checker/action.yml

```diff
@@ -0,0 +1,66 @@
+name: license-checker
+description:
+  "Check for restricted licenses. A wrapper for
+  [LicenseFinder](https://github.com/pivotal/LicenseFinder)"
+
+inputs:
+  # specific inputs
+  license-finder-custom-args:
+    description: "custom args to pass to license_finder"
+    required: false
+    default: |
+      --recursive \
+      --use_spdx_id \
+      --format=markdown \
+      --decisions-file=${{ github.action_path }}/doc/dependency_decisions.yml \
+      --quiet
+  license-finder-version:
+    description: "license_finder version"
+    required: false
+    default: "7.2.1"
+  working-directory:
+    description: "working directory for license_finder"
+    required: false
+    default: "."
+  # grafana inputs (optional)
+  metrics-job-name:
+    description: "grafana metrics job name"
+    required: false
+  metrics-id:
+    description:
+      "grafana metrics id, used for continuity of metrics during job name
+      changes  - required if metrics-job-name is passed"
+    required: false
+  gc-host:
+    description: "grafana hostname - required if metrics-job-name is passed"
+    required: false
+  gc-basic-auth:
+    description: "grafana basic auth - required if metrics-job-name is passed"
+    required: false
+  gc-org-id:
+    description:
+      "grafana org/tenant id - required if metrics-job-name is passed"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: "Install license-finder"
+      run: gem install license_finder -v ${{ inputs.license-finder-version }}
```

Comment:
do you get `gem` for free with gha runner?
or do you need something like this?
https://github.com/ruby/setup-ruby

Reply:
I think it's available but I have to use sudo to install to the system-wide gem dir, which may not be ideal?

Reply:
cool as long as it works :)